### ISSUE TITLE
Grafana 2.0.2 (docker image) doesn't show data sources when auth.anonymous enabled and logged in as admin

### ISSUE THREAD

**Issue by discordianfish** [grafana/grafana#2159]
Hi,

if auth.anonymous is enabled, there is no way to add data sources. Even after logging in as admin, the data sources aren't showing up. Another thing I realize is that I got permission denied when trying to create a new org, so it seems even though I'm logged in as admin, the unprivileged/anonymous permissions apply.

I'm using the official grafana image with 2.0.2 installed.

**Comment by torkelo** [comment]
What did you configure anonymous org  role to? You need org role of Admin to view/edit data sources

**Comment by discordianfish** [comment]
@torkelo I kept all default for the anonymous stuff beside enabling it. But I logged in as admin and still couldn't see the data sources.

**Comment by damm** [comment]
```
[auth.anonymous]
# enable anonymous access
enabled = false

# specify organization name that should be used for unauthenticated users
org_name = Main Org.

# specify role for unauthenticated users
org_role = Viewer
```

Change the `org_role` and win right? also pick the right org name to start with :) or it messes with anonymous mode.

**Comment by damm** [comment]
I only currently set the `org_role` to editor and use anonymous mode.  The thought of giving admin to anonymous is :( scary

**Comment by torkelo** [comment]
@discordianfish can you show more of you config and what you did? You cant edit data sources even when logging in with the default admin user? What role does your admin have for the current organization? Did you remove the default admin and create a new one?

**Comment by discordianfish** [comment]
Looks like there is still some confusion. I don't want to give anonymous users admin permissions. I simply enabled auth.anonymous. After that, I logged in as admin but couldn't see the data source. I disabled anonymous auth again, then logged in and the data sources where there. I added a data source and just re-enable anonymous auth. Now I can see the data sources just fine.
I quickly tried to reproduce it but couldn't. It might be related to using postgres and starting without data sources. Anyway, feel free to keep this issue closed and I'll update if I should run into this again.
